lib/trailblazer/activity/testing.rb

```diff
@@ -24,21 +25,22 @@ def self.def_steps(*names)
     # @example
     #   task task: T.def_task(:create)
     def self.def_task(name)
-      Module.new do
-        define_singleton_method(name) do | (ctx, flow_options), ** |
+      Module.new {
```

Comment:
What is the "best practice" here? I thought `do ... end` when multiline?

Reply:
we can do that. i just have to update rubocop rules to ignore it.